File: examples/univariate/example_binomtest.py
from statsmodels.stats.proportion import binom_test
from statsmodels.stats.proportion import proportions_ztest
Boston = ghload("Boston200.rds")
res = binom_test(sum(Boston['chas']), len(Boston['chas']))
print(res)
#
res = proportions_ztest(sum(Boston['chas']), len(Boston['chas']), 
                        value=0.5)
print(res)

File: examples/univariate/example_binomtest.R
Boston <- mmstat4::ghload("Boston200.rds")
# Sign test
# H0: pi = pi0
# H1: pi <> pi0
binom.test(sum(Boston$chas==1), length(Boston$chas))
prop.test(sum(Boston$chas==1), length(Boston$chas))
